ci/docker/deploy-amdgpu.Dockerfile

```diff
@@ -70,7 +70,7 @@ RUN if [ "$USE_MKL" = "ON" ]; then \
 # Deploy Extra RocBlas files separately.
 ARG USE_ROCBLAS=OFF
 RUN if [ "$USE_ROCBLAS" = "ON" ]; then \
-      cp -r `spack -e ci location -i rocblas`/lib/library ${DEPLOY}/usr/lib ; \
+      cp -r `spack -e ci location -i rocblas`/lib/rocblas/library ${DEPLOY}/usr/lib ; \
```

Comment:
Did something change here in one of the last pushes? CI is now again complaining that this doesn't exist: https://gitlab.com/cscs-ci/eth-cscs/DLA-Future/-/jobs/3118583844#L1449.